Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -997,6 +1000,24 @@ namespace GradientSignal
         return m_imageDescriptor.m_size.m_width;
     }
 
+    AZ::Vector2 ImageGradientComponent::GetImagePixelsPerMeter() const
+    {
+        // Get the number of pixels in our image that maps to each meter based on the tiling settings.
+
+        const auto& width = m_imageDescriptor.m_size.m_width;
+        const auto& height = m_imageDescriptor.m_size.m_height;
+
+        if (width > 0 && height > 0)
+        {
+            const AZ::Aabb bounds = m_gradientTransform.GetBounds();
+            const AZ::Vector2 boundsMeters(bounds.GetExtents());
+            const AZ::Vector2 imagePixelsInBounds(width / GetTilingX(), height / GetTilingY());
+            return imagePixelsInBounds / boundsMeters;
+        }
+
+        return AZ::Vector2(0.0f, 0.0f);
```

Comment:
Suggestion: could we use `AZ::Vector2::CreateZero()` instead? I know it's a bit more verbose but potentially more idiomatic?